test/e2e/testdata/external_secret.yaml

```diff
@@ -5,4 +5,9 @@ metadata:
     app.kubernetes.io/name: cluster
     app.kubernetes.io/managed-by: external-secrets-operator-e2e
   name: cluster
-spec: {}
+spec:
+  networkPolicies:
+    - name: allow-external-secrets-egress
+      componentName: ExternalSecretsCoreController
+      egress:
+        - {} # Allow all egress for external issuers communication
```

Comment:
Let us update it to specific provider, this will help with the tests, instead of allowing all, equivalent to not having NP.